content/en/docs/reference/access-authn-authz/abac.md

```diff
@@ -8,6 +8,7 @@ title: Using ABAC Authorization
 content_template: templates/concept
 weight: 80
 ---
+(NOTE: This ABAC Authorization feature has been deprecated )
```

Comment:
```suggestion
{{< note >}}This ABAC Authorization feature has been deprecated{{< /note >}}
```

We have a Hugo shortcode that lets you call out a note.

Reply:
done